src/ssl/ensemble.py

```diff
@@ -20,9 +23,18 @@ def add_classifier(self, classifier):
         Args:
             classifier: Classificador
         """
-        flexconc = FlexConC(classifier)
+        flexconc = self.ssl_algorithm(classifier, **self.params)
         self.ensemble.append(flexconc)
 
+    def add_model(self, model):
```

Comment:
```suggestion
    def add_model(self, model) -> None:
```
Add tipo do argumento.

Reply:
feito